queries.sql

```diff
@@ -36,3 +36,41 @@ SELECT neutered,MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-12-31' AND '2000-12-31' GROUP BY species;
 
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Angemon','Boarmon');
+
+SELECT animals.name
+FROM animals
+JOIN owners ON owners.id = animals.owner_id 
+WHERE owners.full_name = 'Melody Pond';
```

Comment:
- [ ] The query provided and the one in the screenshot are two different queries. Kindly update either your screenshot or the query.

**This is the one in the screenshot:**

![image](https://user-images.githubusercontent.com/86318284/213213168-c90546fc-4e9c-43e3-bd90-cbccacab6911.png)